great_expectations_cloud/agent/agent.py

```diff
@@ -182,19 +186,19 @@ def _listen(self) -> None:
         try:
             client = AsyncRabbitMQClient(url=str(self._config.connection_string))
             subscriber = Subscriber(client=client)
-            print("The GX Agent is ready.")
+            LOGGER.info("The GX Agent is ready.")
             # Open a connection until encountering a shutdown event
             subscriber.consume(
                 queue=self._config.queue,
                 on_message=self._handle_event_as_thread_enter,
             )
         except KeyboardInterrupt:
-            print("Received request to shut down.")
+            LOGGER.debug("Received request to shut down.")
         except (SubscriberError, ClientError):
-            print("The connection to GX Cloud has encountered an error.")
+            LOGGER.exception("The connection to GX Cloud has encountered an error.")
         except GXAgentUnrecoverableConnectionError:
-            print("The connection to GX Cloud has encountered an unrecoverable error.")
             sys.exit(1)
+            LOGGER.exception("The connection to GX Cloud has encountered an unrecoverable error.")
```

Comment:
Consider moving this above the `sys.exit(1)` call.

Reply:
Oops, thank you!